layouts/html.hbs

```diff
@@ -136,13 +134,19 @@
           iframeLoadedResolve();
         },
         onMessage: (message) => {
-          if (message.token) {
-            initAnswersJWT(message.token);
-            return;
+          if (message.runtimeConfig) {
+            HitchhikerJS.RuntimeConfigReceiver.handle(message.runtimeConfig);
+          } else if (window.isOverlay) {
+            window.Overlay.onMessage(message);
           }
-          window.isOverlay && window.Overlay.onMessage(message);
         }
       };
+      {{#if global_config.initializeManually}}
+        const initializeManually = true;
+      {{else}}
+        const initializeManually = false;
+      {{/if}}
+      const experienceInitializer = new HitchhikerJS.ExperienceInitializer(initializeManually).setup();
```

Comment:
Do we only need an `experienceInitializer` in the `initializeManually` case or no?

Reply:
Currently it's used for both automatic and manual initialization. I could re-work it so that the automatic initialization occurs in layouts/html.hbs, and the experience initializer is just for initializeManually. (I would also probably update the naming). That might be a better separation of concerns

Reply:
Got it, I'm fine either way!